packages/boot/test/bootstrapTests/test-liquidation-concurrent-2b.ts

```diff
@@ -0,0 +1,486 @@
+import { ExecutionContext, TestFn } from 'ava';
+import {
+  LiquidationTestContext,
+  addSTARsCollateral,
+  makeLiquidationTestContext,
+  scale6,
+} from './liquidation.ts';
+
+const test = anyTest as TestFn<LiquidationTestContext>;
+
+const atomSetup = {
+  vaults: [
+    {
+      atom: 15,
+      ist: 100,
+      debt: 100.5,
+    },
+    {
+      atom: 15,
+      ist: 103,
+      debt: 103.515,
+    },
+    {
+      atom: 15,
+      ist: 105,
+      debt: 105.525,
+    },
+  ],
+  bids: [
+    {
+      give: '25IST',
+      discount: 0.3,
+    },
+    {
+      give: '75IST',
+      discount: 0.22,
+    },
+  ],
+  price: {
+    starting: 12.34,
+    trigger: 9.99,
+  },
+  auction: {
+    start: {
+      collateral: 45,
+      debt: 309.54,
+    },
+    end: {
+      collateral: 31.414987,
+      debt: 209.54,
+    },
+  },
+};
+
+const starsSetup = /** @type {const} */ {
+  vaults: [
+    {
+      atom: 15,
+      ist: 110,
+      debt: 110.55,
+    },
+    {
+      atom: 15,
+      ist: 112,
+      debt: 112.56,
+    },
+    {
+      atom: 15,
+      ist: 113,
+      debt: 113.565,
+    },
+  ],
+  bids: [
+    {
+      give: '25IST',
+      discount: 0.3,
+    },
+    {
+      give: '75IST',
+      discount: 0.22,
+    },
+  ],
+  price: {
+    starting: 13.34,
+    trigger: 10.99,
+  },
+  auction: {
+    start: {
+      collateral: 45,
+      debt: 336.675,
+    },
+    end: {
+      collateral: 32.65,
+      debt: 236.675,
+    },
+  },
+};
+
+const setups = {
+  ATOM: atomSetup,
+  STARS: starsSetup,
+};
+
+const atomOutcome = /** @type {const} */ {
+  bids: [
+    {
+      payouts: {
+        Bid: 0,
+        Collateral: 10.01,
+      },
+    },
+    {
+      payouts: {
+        Bid: 0,
+        Collateral: 3.575,
+      },
+    },
+  ],
+  reserve: {
+    allocations: {
+      ATOM: 1.619207,
+    },
+    shortfall: 5.525,
+  },
+  vaultsActual: [
+    {
+      debt: 100.5,
+      locked: 14.998993,
+    },
+    {
+      debt: 103.515,
+      locked: 14.998963,
+    },
+    {
+      locked: 0,
+    },
+  ],
+};
+
+const starsOutcome = /** @type {const} */ {
+  bids: [
+    {
+      payouts: {
+        Bid: 0,
+        Collateral: 9.099,
+      },
+    },
+    {
+      payouts: {
+        Bid: 0,
+        Collateral: 3.2497,
+      },
+    },
+  ],
+  reserve: {
+    allocations: {
+      ATOM: 2.87192,
+    },
+    shortfall: 13.565,
+  },
+  vaultsActual: [
+    {
+      debt: 100.5,
+      locked: 14.998993,
+    },
+    {
+      debt: 103.515,
+      locked: 14.998963,
+    },
+    {
+      locked: 0,
+    },
+  ],
+};
+
+const outcomes = {
+  ATOM: atomOutcome,
+  STARS: starsOutcome,
+};
+
+test.before(async t => {
+  t.context = await makeLiquidationTestContext(t);
+  await addSTARsCollateral(t);
+});
+test.after.always(t => {
+  return t.context.shutdown && t.context.shutdown();
+});
+
+const checkFlow2b = async (t: ExecutionContext<LiquidationTestContext>) => {
```

Comment:
where is the "b" coming from?

Reply:
Flow 2b from https://github.com/Agoric/agoric-sdk/issues/7123

Reply:
I asked socratically :)  (that 2b is for a different test flow than this one)

Reply:
I asked socratically :)  (that 2b is for a different test flow than this one)